fastlane_core/lib/fastlane_core/string_filters.rb

```diff
@@ -1,15 +1,15 @@
 class String
   # Truncates a given +text+ after a given <tt>length</tt> if +text+ is longer than <tt>length</tt>:
   #
-  #   'Once upon a time in a world far far away'.truncate(27)
+  #   'Once upon a time in a world far, far away'.truncate(27)
   #   # => "Once upon a time in a wo..."
```

Comment:
the `#` side doesn't care, but this is the correct way to write `far, far away`

There should also be a `,` after `time`, but that would change the expected output :)

Reply:
I wouldn't be opposed to changing it so that there's a comma after `time` and then also updating the output :P it's just sample code in docs anyway haha shouldn't break anything